README.md

```diff
@@ -131,6 +131,33 @@ For more guidance in running `CRI-O`, visit our [tutorial page](tutorial.md)
 [libpod-hooks]: https://github.com/containers/libpod/blob/v0.6.2/pkg/hooks/README.md
 [spec-hooks]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#posix-platform-hooks
 
+#### The HTTP API
```

Comment:
We can probably add a statement saying that this isn't version and is likely to change.

Reply:
Yep, added an additional comment about that fact.